model/search/GenerisSearch.php

```diff
@@ -51,7 +51,10 @@ public function query($queryString, $type, $start = 0, $count = 10, $order = 'id
         ]);
         $ids = array();
         foreach ($results as $resource) {
-            $ids[] = $resource->getUri();
+            $ids[] = [
```

Comment:
do we need to update tests ?

Reply:
haven't found any affected ones

Reply:
We should have merged to integration branch till Advanced search is stable @voknil